lib/src/presentation/lit_auth_ui.dart

```diff
@@ -46,9 +46,9 @@ class LitAuthState extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    final user = context.watch<User>();
+    final user = context.watch<LitUser>();
     return user.when(
-      (user) => authenticated,
+      (dynamic user) => authenticated,
```

Comment:
Scratch the dynamic here

Reply:
Without the dynamics, the compiler complained